KeystreamGenerator.rb

```diff
@@ -0,0 +1,138 @@
+class KeystreamGenerator
+
+  attr_reader :keystream_length, :deck
+
+  def initialize(keystream_length)
+    @keystream_length = keystream_length
+  end
+
+  def generate_keystream
+    @deck = build_deck
+    generate_letters
+  end
+
+  private
+
+  NUMBER_CONVERTER = Hash[(1..26).zip ('A'..'Z')]
```

Comment:
Two things:

1) Constants usually go up at the top of the class
2) This is a Value Object which could be a separate class. That way, you can expand the whole hash and it would be easier to understand what it's doing.